### DISCUSSION TITLE
How to access camera position

### DISCUSSION THREAD

**Discussion by tkulpa** [rnmapbox/maps#2988] [Q&A]
I need to save camera position to restore it later and couldn't access camera position & zoom from camera ref. Is there any other way than listening to `onCameraChanged` to get current camera position?

**Comment by Crysp** [comment]
Similar question. Why information that relates to camera is handled inside another component?

I want to create HOC for camera and it is not so easy as I expect. When user changes camera position I should forward this data from MapView to Camera
